src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+
+    public static void printWelcome() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        printDividerLine();
+        System.out.println("Hello! I'm Duke! \n " + logo);
+        System.out.println("What can I do for you?");
+        printDividerLine();
+
+    }
+
+    public static void printDividerLine() {
+        System.out.println("___________________________________ \n");
+    }
+
+    public static void printGoodbye() {
+        printDividerLine();
+        System.out.println("Bye. Hope to see you again soon! :)");
+        printDividerLine();
+    }
+
+
+    public static void main(String[] args) {
+        Scanner userResponseScanner = new Scanner(System.in);
+        boolean quitResponseLoop = false;
+        //String userResponse;
+        String command;
+        String commandDetails = null;
+        ArrayList<Task> taskList = new ArrayList<>();
+
+        printWelcome();
+
+        while (!quitResponseLoop) {
+            command = userResponseScanner.nextLine().trim();
+
+            if(command.contains(" ")) {
+                String[] splitUserResponse = command.split(" ", 2);
+                command = splitUserResponse[0];
+                commandDetails = splitUserResponse[1];
+            }
```

Comment:
Bug here. If command does not contain any spaces, it falls through all the branches and commandDetails is still null.